src/Resource/Collection/ArrayResourceCollection.php

```diff
@@ -40,7 +40,7 @@ class ArrayResourceCollection implements IteratorAggregate, ResourceCollection
      * You can pass the resources that you want to initially store in the
      * collection as argument.
      *
-     * @param PuliResource[] $resources The resources to store in the collection.
+     * @param PuliResource[]|\Traversable $resources The resources to store in the collection.
```

Comment:
Can you change this to `Traversable` (intead of `\Traversable`) and import the class instead? Also please add this as well for `replace()` and `merge()` (in the `ResourceCollection` interface).